virttest/env_process.py

```diff
@@ -1240,6 +1240,25 @@ def preprocess(test, params, env):
             test.cancel("Got host qemu version:%s, which is not in %s" %
                         (host_qemu, required_qemu))
 
+    # Checking required virtio-win version, if not satisfied, cancel test
+    if params.get("required_virtio_win"):
+        if params.get("cdrom_virtio"):
+            required_virtio_win = params.get("required_virtio_win")
```

Comment:
We don't provide a default `required_virtio_win` so here could be `params['']`, as `params.get()` has more trasactions which is not necessary here.